*** xorg-server-1.16.4/xkb/xkbInit.c.orig	2015-02-10 18:32:06.000000000 -0500
--- xorg-server-1.16.4/xkb/xkbInit.c	2015-03-07 03:00:02.771375526 -0500
***************
*** 560,565 ****
--- 560,569 ----
      if (xkb_cached_map)
          LogMessageVerb(X_INFO, 4, "XKB: Reusing cached keymap\n");
      else {
+         char *xkbBinDir = getenv("XKB_BINDIR");
+         if (xkbBinDir)
+             XkbBinDirectory = Xstrdup(xkbBinDir);
+ 
          if (rmlvo)
              xkb_cached_map = XkbCompileKeymap(dev, rmlvo);
          else

